libs/template/presets/storefront/experience/pages/registration-page.ts

```diff
@@ -0,0 +1,31 @@
+import { ExperienceComponent } from '@spryker-oryx/experience';
+
+export const registrationPage: ExperienceComponent = {
```

Comment:
this is considered a breaking change:
- customer might already have their own at the same route
- a customer that would get this page, would not be prepared for this, leading to operation issues, broken integrations, misalignment of UI, etc.

Can we wrap it inside the latest feature flag?